SimpleWAWS/Code/ResourcesManager.cs

```diff
@@ -269,7 +269,7 @@ private void DeleteResourceGroup(ResourceGroup resourceGroup)
                         //Enable ZRay
                         await site.EnableZRay(resourceGroup.GeoRegion);
                     }
-                    else if (((template.Language!=null) && template.Language.Equals("NodeJs", StringComparison.OrdinalIgnoreCase))|| template.Name.Equals("ExpressJs", StringComparison.OrdinalIgnoreCase))
+                    else if (((template.Language!=null) && template.Language.Equals("NodeJs", StringComparison.OrdinalIgnoreCase))|| template.Name.Equals("Express", StringComparison.OrdinalIgnoreCase))
```

Comment:
I'd prefer not to make any other refactorings for this change, since I'm sure there are other simplifications that could be done in this file.